exporter/elasticsearchexporter/factory.go

```diff
@@ -37,7 +40,7 @@ func NewFactory() exporter.Factory {
 
 func createDefaultConfig() component.Config {
 	qs := exporterhelper.NewDefaultQueueSettings()
-	qs.Enabled = false
+	qs.NumConsumers = 100 // default is too small as it also sets batch sender concurrency limit
```

Comment:
Worth linking to https://github.com/open-telemetry/opentelemetry-collector/issues/10368 here?